Aquality.Selenium/src/Aquality.Selenium/Browsers/DevToolsEmulationExtensions.cs

```diff
@@ -0,0 +1,167 @@
+﻿using Aquality.Selenium.Core.Utilities;
+using OpenQA.Selenium.DevTools;
+using OpenQA.Selenium.DevTools.V85.Emulation;
+using System;
+using System.Threading.Tasks;
+
+namespace Aquality.Selenium.Browsers
+{
+    /// <summary>
+    /// Implementation of version-independent emulation DevTools commands as extensions for <see cref="DevToolsHandling"/>.
+    /// </summary>
+    public static class DevToolsEmulationExtensions
+    {
+        /// <summary>
+        /// Tells whether emulation is supported.
+        /// </summary>
+        /// <param name="devTools">Current instance of <see cref="DevToolsHandling"/>.</param>
+        /// <returns>A task for asynchronous command with boolean result.</returns>
+        public static async Task<bool> CanEmulate(this DevToolsHandling devTools)
+        {
+            var response = await devTools.SendCommand(new CanEmulateCommandSettings());
```

Comment:
Could you please say, do I understand right that we don't use code convention about var in the project? (https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#implicitly-typed-local-variables)
![image](https://user-images.githubusercontent.com/15047073/155411714-af05992d-fafa-40a3-8d50-726e58c85e84.png)

Reply:
yes, we did not follow this recommendation in our projects yet. We may decide to follow it in the future, then we'll need to refactor the whole library. Definitely not inside of this PR